extensions/amp-story/1.0/page-advancement.js

```diff
@@ -462,10 +462,7 @@ export class ManualAdvancement extends AdvancementConfig {
       (el) => {
         tagName = el.tagName.toLowerCase();
 
-        if (
-          tagName === 'amp-story-page-attachment' ||
-          tagName === 'amp-story-page-outlink'
-        ) {
+        if (el.classList.contains('amp-story-draggable-drawer-root')) {
```

Comment:
No need to revert, I'm just trying to offer an alternative argument to consider next times we want to do such optimizations.
The previous implementation had the advantage of being much easier to understand for people with 0 context, making the codebase a bit more accessible. It used to read as "content within page-attachment or page-outlink don't trigger page advancement". Now with this class it's a bit less obvious.

Reply:
Agree that this needs a comment. Opened #36764